src/Extension/CoreExtension.php

```diff
@@ -1478,7 +1478,13 @@ function twig_get_attribute(Environment $env, Source $source, $object, $item, ar
 
     // object property
     if (/* Template::METHOD_CALL */ 'method' !== $type) {
-        if (isset($object->$item) || \array_key_exists((string) $item, (array) $object)) {
+//        if (isset($object->$item) || \array_key_exists((string) $item, (array) $object)) {
+        /**
+         * NOTE: this change is added fix issue when ActiveRecord object has the property but its value is null. e.g booking.agentId.
```

Comment:
A comment saying `this change` is totally useless, as the comment is not describing a change (commit messages do that) but the code in the class.

And keeping the old line as a comment is also totally useless (keeping the history is handled by git in a much better way)

Reply:
Thanks @stof the feedback. I have removed it.